recipes/conversation_view/message_input/message_input.vue

```diff
@@ -0,0 +1,379 @@
+<template>
+  <div>
+    <!-- TODO: A purpose-built place to display validation errors using DtBanner -->
+    <div
+      class="d-d-flex d-fd-column d-bar8 d-baw1 d-ba d-c-text"
+      :class="{ 'd-bc-black-500 d-bs-sm': hasFocus, 'd-bc-default': !hasFocus }"
+    >
+      <!-- Some wrapper to restrict the height and show the scrollbar -->
+      <div class="d-of-auto d-mx16 d-mt8 d-mb4 d-hmx40p">
+        <dt-rich-text-editor
+          v-model="inputValue"
+          :editable="editable"
+          :input-aria-label="inputAriaLabel"
+          :input-class="inputClass"
+          :output-format="outputFormat"
+          :auto-focus="autoFocus"
+          :link="link"
+          :placeholder="placeholder"
+          v-bind="$attrs"
+          @focus="hasFocus = true"
+          @blur="hasFocus = false"
+        />
+      </div>
+      <!-- @slot Slot for attachment carousel -->
+      <slot name="middle" />
+      <!-- Section for the bottom UI -->
+      <section class="d-d-flex d-jc-space-between d-mx8 d-my4">
+        <!-- Left content -->
+        <div class="d-d-flex">
+          <dt-button
+            size="sm"
+            circle
+            importance="clear"
+          >
```

Comment:
These buttons need aria-label, will have to be a prop because of i18n

Reply:
got it.